docs/topic_guides/blocking/blocking_rules.md

```diff
@@ -0,0 +1,65 @@
+
+Considering a dataset of 1 million records, comparing each record against all of the other records in the dataset generates ~500 billion pairwise comparisons. However, we know the vast majority of these record comparisons won't be matches, so processing the full ~500 billion comparisons would be largely pointless (as well as costly and time-consuming).
+
+Instead, we can define a subset of potential comparisons using **Blocking Rules**. These are rules that define "blocks" of comparisons that should be considered. For example, the blocking rule:
+
+`"l.first_name = r.first_name and l.surname = r.surname"` 
+ 
+will generate pairwise record comparisons amongst pairwise comparisons where first name and surname match.
+
+Within a Splink model, you can specify multiple "blocks" through multiple Blocking Rules to ensure all potential matches are considered.
+
+???+ "Further Reading"
+
+    For more information on blocking, please refer to [this article](https://toolkit.data.gov.au/data-integration/data-integration-projects/probabilistic-linking.html#key-steps-in-probabilistic-linking)
+
+### Choosing Blocking Rules
+
+The blocking process is a compromise between the amount of **compuational resource** used when comparing records and **capturing all true matches**. 
+
+Even after blocking, the number of comparisons generated is usually much higher than the number of input records - often between 10 and 1,000 times higher. As a result, the performance of Splink is heavily influenced by the number of comparisons generated by the blocking rules, rather than the number of input records.
+
+Getting the balance right between compuational resource and capturing matches can be tricky, and is largely dependent on the specific datasets and use case of the linkage. In general, we recommend a strategy of starting with strict blocking rules, and gradually loosening them. Sticking to less than 10 million comparisons is a good place to start, before scaling jobs up to 100s of millions (:simple-duckdb: DuckDB on a laptop), or sometimes billions (:simple-apachespark: Spark or :simple-amazonaws: Athena). 
+ 
+Guidance for choosing Blocking Rules can be found in the two [Blocking in Splink](#blocking-in-splink) topic guides.
+
+!!! note "Taking blocking to the extremes"
+    If you have a large dataset to deduplicate, let's consider the implications of two cases of taking blocking to the extremes:
+
+    **Not enough blocking** (ensuring all matches are captured)  
+    There will be too many record pairs to consider, which will take an extremely long time to run (hours/days) or the process will be so large that it crashes.
+
+    **Too much blocking** (minimising computational resource)  
+    There won't be enough records pairs to consider, so the model won't perform well (or will struggle to be trained at all). 
+
+
+## Blocking in Splink
+
+There are two areas in Splink where blocking is used:
+
```

Comment:
Maybe 'making predictions' should come first here, since it's the more familiar concept (that will be understood by a wider audience).

Might even be worth saying this explicitly in the text - something like: 

There are two areas in Splink where blocking is used:

- The first is to generate pairwise comparisons when finding links (running `predict()`).  This is the sense in which 'blocking' is usually understood in the context of record linkage

- The second is a less familiar application of blocking: using it for the purpose of model training.

each of which is described in their own, dedicated topic guide.

Reply:
Implemented